packages/shared/lib/core/wallet/utils/getDefaultTransactionOptions.ts

```diff
@@ -1,16 +1,18 @@
-import { Bip44Address, TransactionOptions } from '@iota/sdk/out/types'
+import { AddressType, TransactionOptions } from '@iota/sdk/out/types'
 import { getSelectedWallet } from '../stores'
 
-export function getDefaultTransactionOptions(address?: string): TransactionOptions | undefined {
-    if (!address) {
+export function getDefaultTransactionOptions(accountId?: string): TransactionOptions | undefined {
+    if (!accountId) {
         const wallet = getSelectedWallet()
         if (!wallet) return
-        address = wallet.depositAddress
+        accountId = wallet.mainAccountId
     }
-    const value: Bip44Address = {
-        address: address,
-        keyIndex: 0,
-        internal: false,
+
+    // TODO: update interface when https://github.com/iotaledger/iota-sdk/issues/1975 is merged
```

Comment:
maybe we should create an issue for this and reference this comment :pray:

Reply:
[Ready](https://github.com/iotaledger/firefly/issues/8041)